packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -2501,3 +2502,40 @@ class ToCsvButton extends StatelessWidget {
     );
   }
 }
+
+class CopyButton extends StatelessWidget {
```

Comment:
we already have a common widget for the copy control. `CopyToClipboardControl` displays a notification upon success too. See if we can reuse that.